terraform/registry/bucket.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_s3_bucket" "sunodo_bucket" {
+  bucket = "${var.bucket_name}"
+  force_destroy = false
+
+  tags = {
+    Name        = "sunodo_bucket"
+    Environment = "Dev"
+  }
+}
+
+resource "aws_s3_bucket_acl" "sunodo_bucket_acl" {
+  bucket = aws_s3_bucket.sunodo_bucket.id
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_public_access_block" "sunodo_bucket_block_pub_access" {
+    bucket = aws_s3_bucket.sunodo_bucket.id
+    block_public_acls   = true
+    block_public_policy = true
+    restrict_public_buckets = true
```

Comment:
Here you use 4 spaces, and 2 elsewhere

Let's stick to a pattern. 

Is there any linter for HCL that we could use within VSCode to make this formating automatic?

Reply:
Yes, I know there is a TFLint that can lint our TF configs and we even can configure GH action for it ...

Reply:
VSCODE extension : https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform